src/components/PrimitiveCheckbox/styled/innerContainer.js

```diff
@@ -60,21 +79,6 @@ const StyledInnerContainer = attachThemeAttrs(styled.div)`
         border-color: ${props => props.palette.border.divider};
     }
 
-    [type='checkbox']:indeterminate + .rainbow-table-input-checkbox_faux::after,
-    [type='checkbox']:indeterminate ~ .rainbow-table-input-checkbox_faux::after,
-    [type='checkbox']:indeterminate + label .rainbow-table-input-checkbox_faux::after {
-        content: '';
-        display: block;
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        width: 0.5rem;
-        height: 2px;
-        background-color: ${props => props.palette.brand.main};
-        border: 0;
-        transform: translate3d(-50%, -50%, 0);
-    }
```

Comment:
I think using the native indeterminate you don't need to make any change in this file